src/main/scala/com/typesafe/sbt/packager/docker/DockerPlugin.scala

```diff
@@ -80,6 +83,9 @@ object DockerPlugin extends AutoPlugin {
     dockerEntrypoint := Seq("bin/%s" format executableScriptName.value),
     dockerCmd := Seq(),
     dockerExecCommand := Seq("docker"),
+    dockerVersion := Try(Process(dockerExecCommand.value ++ Seq("version --format '{{.Server.Version}}'")).!!)
```

Comment:
Guys, I don't think this works...
I believe it needs to be `Seq("version", "--format", "'{{.Server.Version}}'")`, otherwise your shell will add another set of ticks around the whole thing and Docker will report that it does not know the command.

Reply:
nvm me, already fixed somewhere else :D

Reply:
#1052 and @mrfyda ftw! :D